src/api-binder.ts

```diff
@@ -325,13 +311,15 @@ export class APIBinder {
 			registered_at: Math.floor(Date.now() / 1000),
 		});
 
-		return (await this.balenaApi
-			.post({ resource: 'device', body: device })
-			// TODO: Remove the `as number` when we fix the config typings
-			.timeout(conf.apiTimeout)) as Device;
+		return (
+			(await this.balenaApi
+				.post({ resource: 'device', body: device })
+				// TODO: Remove the `as number` when we fix the config typings
```

Comment:
We can remove this comment now it looks like

Reply:
ack